x/order/keeper/invariant.go

```diff
@@ -17,15 +17,15 @@ func RegisterInvariants(ir sdk.InvariantRegistry, keeper Keeper) {
 // locks amounts held on store
 func ModuleAccountInvariant(keeper Keeper) sdk.Invariant {
 	return func(ctx sdk.Context) (string, bool) {
-		var lockCoins, lockFee, orderLockFee sdk.DecCoins
+		var lockedCoins, lockedFee, orderLockedFee sdk.DecCoins
 
```

Comment:
1. lockedFee, orderLockedFee should be  the plural form, e.g) lockedFees, orderLockedFees. 
2. The following codes are the same.